ugs-core/src/com/willwinder/universalgcodesender/model/GUIBackend.java

```diff
@@ -808,7 +812,33 @@ public void sendMessageForConsole(String msg) {
             //should still send!  Controller probably shouldn't ever be null.
         }
     }
-    
+
+    @Override
+    public void setWorkPosition(Axis axis, double position) throws Exception {
+        controller.setWorkPosition(axis, position);
+    }
+
+    @Override
+    public void setWorkPositionUsingExpression(final Axis axis, final String expression) throws Exception {
```

Comment:
Wow, this is really cool. I had no idea you could do something like this.

Reply:
Me neither, google was my friend! 😄